src/ast/value.rs

```diff
@@ -22,15 +22,17 @@ use std::fmt;
 pub enum Value {
     /// Numeric literal
     #[cfg(not(feature = "bigdecimal"))]
-    Number(String),
+    Number(String, bool),
```

Comment:
nit: I wonder if a new type should have been added rather than change this type?